.github/workflows/CI-e4s.yml

```diff
@@ -0,0 +1,149 @@
+name: CI-e4s
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+concurrency:
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+
+jobs:
+  HIP:
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        cxx: ['hipcc']
+        cmake_build_type: ['Debug', 'Release']
+        kokkos_ver: ['3.6.01']
+        arborx: ['OFF']
+        heffte: ['OFF' ]
+        hypre: ['OFF']
+        liball: ['OFF']
+        silo: ['OFF']
+        hdf5: ['OFF']
+    runs-on: ubuntu-20.04
+    container: docker.io/ecpe4s/e4s-base-rocm:23.05
+    steps:
+      - name: Checkout GTest
+        uses: actions/checkout@v3
+        with:
+          repository: google/googletest
+          ref: release-1.11.0
+          path: gtest
+      - name: Build gtest
+        working-directory: gtest
+        run: |
+          cmake -B build \
+            -DCMAKE_INSTALL_PREFIX=$HOME/gtest \
+            -DCMAKE_BUILD_TYPE=${{ matrix.cmake_build_type }}
```

Comment:
I would always go for Release for GTest.

Reply:
Done